cmake/MacroHelper.cmake

```diff
@@ -213,7 +213,7 @@ macro(nocmodl_mod_to_cpp modfile_basename)
       ${CMAKE_COMMAND} -E env "MODLUNIT=${PROJECT_BINARY_DIR}/share/nrn/lib/nrnunits.lib"
       ${NRN_NOCMODL_SANITIZER_ENVIRONMENT} $<TARGET_FILE:nocmodl>
       ${PROJECT_SOURCE_DIR}/${modfile_basename}.mod
-    COMMAND sed "'s/_reg()/_reg_()/'" ${PROJECT_SOURCE_DIR}/${modfile_basename}.cpp >
+    COMMAND sed "s/_reg\\(\\)/_reg_\\(\\)/" ${PROJECT_SOURCE_DIR}/${modfile_basename}.cpp >
```

Comment:
Still needs some massaging :rage:

Reply:
more than this? 🤔

Reply:
Yeah, that was too much escaping for :window: :window: